src/header/header.scss

```diff
@@ -31,6 +31,10 @@
     flex: 2 1 33%;
     @include iui-header-buttons;
 
+    > nav {
+      display: contents;
+    }
```

Comment:
This is the part I'm not sure about. Do you think there is a better way?
@bentleyvk @veekeys

Reply:
tried flex?

Reply:
This one worked for me...

![image](https://user-images.githubusercontent.com/81580355/118505114-cbc80d80-b734-11eb-9162-d215d967c9c0.png)

Reply:
and what about the selector itself? `> nav` is ok?

Reply:
Yeah I guess... as we do not want to have a class for it now..

Reply:
Pushed